src/bundle/Controller/ContentViewController.php

```diff
@@ -525,6 +528,45 @@ static function (Repository $repository) use ($contentInfo): bool {
 
         $view->addParameters(['content_has_reverse_relations' => $hasReverseRelations]);
     }
+
+    /**
+     * @param \Ibexa\Core\MVC\Symfony\View\ContentView $view
+     */
+    private function supplyUserInvitation(ContentView $view): void
+    {
+        $content = $view->getContent();
+        $contentType = $this->contentTypeService->loadContentType(
+            $view->getContent()->contentInfo->contentTypeId,
+            $this->userLanguagePreferenceProvider->getPreferredLanguages()
+        );
+        $contentIsUserGroup = (new ContentTypeIsUserGroup($this->configResolver->getParameter('user_group_content_type_identifier')))
+            ->isSatisfiedBy($contentType);
+        $canSendInvitation = $this->permissionResolver->canUser(
+            'user',
+            'invite',
+            $content
+        );
+
+        if ($contentIsUserGroup && $canSendInvitation) {
+            $userInvitation = $this->sfFormFactory->create(
+                UserInvitationType::class,
+                null,
+                [
+                    'action' => $this->generateUrl(
+                        'ibexa.user.invite.to_group',
+                        [
+                            'userGroupId' => $content->contentInfo->id,
```

Comment:
... and here as well?
```suggestion
                            'userGroupId' => $content->getVersionInfo()->getContentInfo()->id,
```